frontend/pages/events/index.vue

```diff
@@ -20,10 +20,15 @@
         :event="event"
       />
     </div>
+    <div class="pt-3 pb-6 space-y-6 md:pt-4 w-1/2">
+      <CardDatePicker></CardDatePicker>
+    </div>
   </div>
 </template>
 
 <script setup lang="ts">
+import CardDatePicker from '~/components/card/CardDatePicker.vue';
```

Comment:
And with Nuxt there's no need to import the components as they're all loaded automatically ✨😊 Types we need to load, but components not :)